[zh-TW]: update CSS "CSS flexible box layout" (#28001)
mdn/translated-content: files/zh-tw/web/css/css_flexible_box_layout/index.md

---

> title: CSS 彈性盒排版

Reviewer: 统一一下翻译，有翻译成排版的，也有翻译成布局的

<img width="264" height="424" alt="image" src="https://github.com/user-attachments/assets/ba0161a2-fae2-4967-8a2b-ef82a2b92265" />

Author: > 统一一下翻译，有翻译成排版的，也有翻译成布局的

我傾向翻譯成排版

Reviewer: 调研一下吧，看业内大多使用哪种翻译

slug: Web/CSS/CSS_flexible_box_layout
l10n:

>   sourceCommit: b8f45350a203be9e6e83c6fcb83c93576d8d5d9c

Reviewer: ```suggestion
  sourceCommit: 0cc9980e3b21c83d1800a428bc402ae1865326b2
```

## 參考

### 屬性

- {{cssxref("align-content")}}
- {{cssxref("align-items")}}
- {{cssxref("align-self")}}
- {{cssxref("flex")}}
- {{cssxref("flex-basis")}}
- {{cssxref("flex-direction")}}
- {{cssxref("flex-flow")}}
- {{cssxref("flex-grow")}}
- {{cssxref("flex-shrink")}}
- {{cssxref("flex-wrap")}}
- {{cssxref("justify-content")}}

### 術語

- {{Glossary("Flexbox")}}
- {{Glossary("Flex container", "彈性容器")}}
- {{Glossary("Flex item", "彈性項目")}}
- {{Glossary("Main axis", "主軸")}}
- {{Glossary("Cross axis", "交錯軸")}}
- {{Glossary("Flex")}}

## 指南

> - [Flexbox 的基本概念](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox)
>   - : Flexbox 功能概覽。
> - [Flexbox 與其他佈局方法的關係](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Relationship_of_flexbox_to_other_layout_methods)
>   - : Flexbox 如何與其他佈局方法及其他 CSS 規範相關聯。

Reviewer: ```suggestion
- [彈性盒的基本概念](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Basic_concepts_of_flexbox)
  - : 彈性盒功能概覽。
- [彈性盒與其他佈局方法的關係](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Relationship_of_flexbox_to_other_layout_methods)
  - : 彈性盒如何與其他佈局方法及其他 CSS 規範相關聯。
```

- [在彈性容器中對齊項目](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Aligning_items_in_a_flex_container)
  - : 盒對齊屬性如何與 Flexbox 一同運作。
- [排序彈性項目](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Ordering_flex_items)
  - : 解釋改變項目順序和方向的不同方法，並涵蓋這樣做可能產生的問題。
- [控制彈性項目沿主軸的比例](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Controlling_ratios_of_flex_items_along_the_main_axis)
  - : 解釋 flex-grow、flex-shrink 和 flex-basis 屬性。
- [精通彈性項目的換行](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Mastering_wrapping_of_flex_items)
  - : 如何建立具有多行的彈性容器，並控制這些行中項目的顯示方式。

> - [Flexbox 的典型用例](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Typical_use_cases_of_flexbox)
>   - : Flexbox 典型的常見設計模式。

Reviewer: ```suggestion
- [彈性盒的典型用例](/zh-TW/docs/Web/CSS/CSS_flexible_box_layout/Typical_use_cases_of_flexbox)
  - : 彈性盒典型的常見設計模式。
```

> - [CSS 佈局：flexbox](/zh-TW/docs/Learn_web_development/Core/CSS_layout/Flexbox)
>   - : 學習如何使用 flexbox 佈局來建立 Web 佈局。
> - [Flexbox 中的盒對齊](/zh-TW/docs/Web/CSS/CSS_box_alignment/Box_alignment_in_flexbox)
>   - : 詳細介紹 [CSS 盒對齊](/zh-TW/docs/Web/CSS/CSS_box_alignment)中專屬於 flexbox 的功能。

Reviewer: ```suggestion
- [CSS 佈局：彈性盒](/zh-TW/docs/Learn_web_development/Core/CSS_layout/Flexbox)
  - : 學習如何使用彈性盒佈局來建立 Web 佈局。
- [彈性盒中的盒對齊](/zh-TW/docs/Web/CSS/CSS_box_alignment/Box_alignment_in_flexbox)
  - : 詳細介紹 [CSS 盒對齊](/zh-TW/docs/Web/CSS/CSS_box_alignment)中專屬於彈性盒的功能。
```
